ops/resources/sql_database/main.tf

```diff
@@ -0,0 +1,51 @@
+resource "azurerm_mssql_server" "octopus" {
+  name                         = "dibbs-ce-${var.env}-octopus-sqlserver"
+  resource_group_name          = var.resource_group_name
+  location                     = var.location
+  version                      = "12.0"
+  administrator_login          = azurerm_key_vault_secret.db_username.value
+  administrator_login_password = data.azurerm_key_vault_secret.db_password.value
+
+  minimum_tls_version = "1.2"
+}
+
+resource "azurerm_mssql_database" "octopus" {
+  name         = "octopus-db"
+  server_id    = azurerm_mssql_server.octopus.id
+  collation    = "SQL_Latin1_General_CP1_CI_AS"
+  license_type = "LicenseIncluded"
+  max_size_gb  = 5
+  sku_name     = "S0"
+  enclave_type = "VBS"
+
+  tags = {
+    foo = "bar"
+  }
+
+  # prevent the possibility of accidental data loss
```

Comment:
Should this be enabled?

Reply:
Enabling this will come as part of #62. I left it as-is for now until we complete a full production rollout and can ensure that no additional changes are needed. Great catch, regardless!